src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -133,8 +133,8 @@ private String getInputWithDates(String sinceDate, String untilDate) {
     }
 
     /**
-     * Generates the testing report and then compared it with the expected report
-     * Re-generates a normal report after the testing finished if the first report is shallow-cloned
+     * Generates the testing report and then compares it with the expected report.
```

Comment:
I think the 'then' in 'then compares' can be dropped.

Reply:
Fixed.